lib/components/widgets/loginScreen/login_screen.dart

```diff
@@ -0,0 +1,134 @@
+                    color: AppColors.homeScreenHeadingColor,
+                    fontSize: screenWidth * 0.08,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                SizedBox(height: screenHeight * 0.02),
+                Text(
+                  'Enter your details below & free log in',
+                  style: TextStyle(
+                    color: AppColors.classroomSecondaryColor,
+                    fontSize: screenWidth * 0.045,
+                  ),
+                ),
+                SizedBox(height: screenHeight * 0.04),
+                // Input Fields
+                Expanded(
+                  flex: isSmallScreen ? 8 : 10,
+                  child: ListView(
+                    physics: isSmallScreen
+                        ? const BouncingScrollPhysics()
+                        : const NeverScrollableScrollPhysics(),
+                    padding: EdgeInsets.zero,
+                    children: [
+                      TextField(
+                        decoration: InputDecoration(
+                          labelText: 'Username',
+                          border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(8),
+                          ),
+                        ),
+                      ),
+                      SizedBox(height: screenHeight * 0.02),
+                      TextField(
+                        obscureText: true,
+                        decoration: InputDecoration(
+                          labelText: 'Password',
+                          border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(8),
+                          ),
+                        ),
+                      ),
+                      SizedBox(height: screenHeight * 0.02),
+                      // Profession Selection
+                      Text(
+                        'Forgot PassWord?',
+                        style: TextStyle(
+                          color: AppColors.homeScreenHeadingColor,
+                          fontSize: screenWidth * 0.045,
+                          fontWeight: FontWeight.bold,
+                        ),
+                        textAlign: TextAlign.end,
+                      ),
+                      SizedBox(height: screenHeight * 0.32),
+                      CustomButton(
+                        buttonText: 'Login',
+                        buttonBgColor: AppColors.btnSecondaryColor,
+                        buttonTextColor: AppColors.white,
+                        onPressed: () {},
+                      ),
+                      SizedBox(height: screenHeight * 0.02),
+                      Center(
+                        child: Text.rich(
+                          TextSpan(
+                            text: "Don't have an account? ",
+                            style: TextStyle(fontSize: screenWidth * 0.035),
+                            children: [
```

Comment:
Instead of hardcoding padding and margin values, consider using constants for sizes and improving responsiveness.